.github/CODEOWNERS

```diff
@@ -56,7 +56,7 @@
 /sdk/cosmos/azure-spring-data-cosmos-test/          @kushagraThapar @mbhaskar @saragluna @yiliuTo @chenrujun @backwind1233 @stliu @zhichengliu12581 @moarychan
 
 # PRLabel: %Digital Twins
-/sdk/digitaltwins/                                  @drwill-ms @azabbasi @johngallardo
+/sdk/digitaltwins/                                  @johngallardo
```

Comment:
Should be > 1 owner to avoid nothing happening during sick/vacation

Reply:
@johngallardo do you have a suggested set of alias's to use here?

Reply:
```suggestion
/sdk/digitaltwins/                                  @johngallardo @efriesner 
```